### PR TITLE
Added basic boostrap styling and navbar

### PR DESCRIPTION
**Pull request by lucascumsille**
Fixes part #176

### CODE REVIEW

File: scoring/static/scss/scoring.scss [hunk 1]
```diff
@@ -57,4 +57,22 @@
 @import "../bootstrap/utilities";
 @import "../bootstrap/print";
 
-@import "mysociety-standard-footer";
+//The variables.scss is not being imported so I'm putting some variables here.
```
Comment (zarino) [inline scoring/static/scss/scoring.scss:60]: @struan can you help @lucascumsille work out what’s going wrong here with the Sass includes? It‘s either something to do with a conflict between caps/scoring filenames and bootstrap filenames, or a conflict between caps and scoring filenames… or we’re just not using the correct relative path when importing.

**Review by zarino — APPROVED** [review]
Overall looks ok. A couple of weird things (oddly specific degrees and percentages in `.bg-grey-gradient`, empty file at `icons.scss`, commented-out code left at the end of `input.scss`) but nothing show-stopping enough to prevent us just merging this in as-is and tidying up later.

File: scoring/static/scss/scoring.scss (hunk 1 above)
Comment (struan) [reply scoring/static/scss/scoring.scss:60]: it's importing fine, some of the colours defined in scoring.scss aren't in variables which seem to be the root of the problem. if you add

```
$color-league-green-l1: #E4FAF5;
$color-league-green-d1: #0BC7A0;
$color-league-blue-l2: #D8F2FE;
$color-league-blue-l1: #7BD2F8;
$color-league-blue-d1: #045C88;
```

to variables then everything seems to work fine.

**Comment by lucascumsille** [comment]
I just added the subheader(Donation strip)
Screenshot 2021-12-22 at 07 13 34

**Review by zarino — APPROVED** [review]
:+1:

**Pull request merged**